Note: Test that should ERROR - moving variable twice

Process called "takes_ownership" takes data as Integer returns Integer:
    Return data
End Process

Process called "main" returns Integer:
    Let x be 42

    Note: First move - this is OK
    Let result1 be takes_ownership(move x)

    Note: ERROR: Cannot move x again - already moved at line above
    Let result2 be takes_ownership(move x)

    Return 0
End Process
